Expensify/App
# Navigate to any report via the URL

# Tests
1. Open the chat application.
1. In another tab, open expensify, create a new report. Add some comments. Copy its reportID.
![image](https://user-images.githubusercontent.com/14356145/91509545-7474cb00-e88f-11ea-96c4-74e1a021f4b8.png)

3. Update the chat URL to `http://localhost:8080/#/<reportID>` and confirm it loads up just fine.
![image](https://user-images.githubusercontent.com/14356145/91754951-2c9bbf80-eb7f-11ea-8d38-5cd03e26f31c.png)


# Fixes
https://github.com/Expensify/ReactNativeChat/issues/278
